doc/build.jl

```diff
@@ -122,6 +122,68 @@ Centrality measures implemented in *LightGraphs.jl* include the following:
 {{maximal_cliques}}
 """
 
+@file "maximumflow.md" """
```

Comment:
I'm not sure we need the entire text here. The nice thing about what @hayd did is that we can use the existing docstrings to generate the doc. Take a look at some of the other entries here for examples.